### DISCUSSION TITLE
Easy responsive images on Github Pages with Cloudinary – Tiberriver256

### DISCUSSION THREAD

**Discussion by giscus** [Tiberriver256/Tiberriver256.GitHub.io#16] [General]
# Easy responsive images on Github Pages with Cloudinary – Tiberriver256

A Github compatible Jekyll include template to produce responsive images using Cloudinary

https://tiberriver256.github.io/jekyll/web%20development/jekyll-cloudinary-github-compatible-snippet/

**Comment by chris4jahn** [comment]
Hi,
I’m looking for an easy solution for responsible images. I’m using GitHub pages and until know don’t understand how to configure Jekyll for this issue. 
Do you use that snippet within every post? Or is this for a central file that is used for every image in the posts. 
In which files do I have to insert that? And how are the resized images used within posts. Could you go into detail with that?
Cheers
Chris
